cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/UserTaskState.java

```diff
@@ -124,7 +124,7 @@ private String getPlaintext(CruiseControlParameters parameters) {
     int userTaskIdLabelSize = 20;
     int clientAddressLabelSize = 20;
     int startMsLabelSize = 20;
-    int statusLabelSize = 10;
+    int statusLabelSize = 20;
```

Comment:
Is it possible to adjust the size dynamically based on the existing `userTaskInfo` -- i.e. in the loop below?